.github/workflows/ci.yml

```diff
@@ -153,12 +153,12 @@ jobs:
             env: { NO_CYTHON_COMPILE: 1, EXCLUDE: "--listfile=tests/pypy_bugs.txt bugs" }
             allowed_failure: true
             extra_hash: "-allowed_failures"
-          # Coverage - Disabled due to taking too long to run
-          # - os: ubuntu-18.04
-          #   python-version: 3.7
-          #   backend: "c,cpp"
-          #   env: { COVERAGE: 1 }
-          #   extra_hash: '-coverage'
+          # Coverage
+          - os: ubuntu-18.04
+            python-version: 3.7
+            backend: "c,cpp"
+            env: { COVERAGE: 1 }
+            extra_hash: '-coverage'
```

Comment:
Isn't this rather redundant with the dedicated `cycoverage` job further down?

Reply:
Idk, it was commented out as "it took too long to finish", I tried it out, now it wors fine, so I left it uncommented